### DISCUSSION TITLE
Main 10@L5.1@High

### DISCUSSION THREAD

**Discussion by SunnyMoo** [cdgriffith/FastFlix#502] [Q&A] [answered]
Hi,
I'm trying to encode using Main 10@L5.1@High but FastFlix keeps on falling back to Main 10@L5.1@Main.
Here's the settings I use:
![fastflix](https://github.com/cdgriffith/FastFlix/assets/50081326/3fdc2200-3c13-4e3f-a00c-5b9b71f0f0c7)
5.1 param is correctly set, as it's 5 by default, but high tier is ignored.
Where do I fail ?
Thanks !

**Comment by cdgriffith** [comment, marked as answer]
The parameter is going through, however the encoder itself will determine if it is going to mark the file as high or main tier.

The lower tier just means the video doesn't use as much bandwidth, so it's playable on more devices. Which is a good thing, so the encoder auto drops tier markings if it can. 

You actually don't need to pass that parameter at all, as it's set that way by default.

**Comment by SunnyMoo** [comment]
Thank you for this explanation, crystal clear.

**Comment by SunnyMoo** [comment]
Update: actually if I don't set _level-idc=5.1:high-tier=1_ param FastFlix does L5@Main by default.
I tested with larger files and those params correctly does L5.1@High, I was wrong in the OP due to bad test file, my bad.
